apps/site/package.json

```diff
@@ -61,22 +68,36 @@
     "tailwindcss": "~4.0.17",
     "unist-util-visit": "~5.0.0",
     "vfile": "~6.0.3",
-    "vfile-matter": "~5.0.1"
+    "vfile-matter": "~5.0.1",
+    "cross-env": "7.0.3",
+    "@types/node": "22.14.0",
```

Comment:
This doesn't match the minor version in `.nvmrc` (afaik `@types/node` should match on major/minor, but patch does not match)